simulate_agent_paths/AgentLocationSimulator.java

```diff
@@ -0,0 +1,77 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Sends post requests to "https://fos-tracker-278709.an.r.appspot.com/update_agent_location"
+ * for updating latitude and longitude of agent in database whose coordinates are
+ * given in the CoordinatesFile.
+ */
+class ThreadforSendingLocationofAgent extends Thread{
```

Comment:
Nit: O capital

Reply:
btw, nit means nitpicking 
it is not compulsory but recommended